src/w3d/renderer/motchan.cpp

```diff
@@ -410,11 +428,11 @@ int TimeCodedBitChannelClass::Get_Bit(int frame)
 
     unsigned int count = 0;
 
-    if (frame >= (m_bits[m_cachedIdx] & 0x7FFFFFFF)) {
+    if (frame >= Get_Frame_From_Data(m_bits[m_cachedIdx])) {
```

Comment:
Doesn't this still throw compiler warning because we still compare `int` with `unsigned int` ?

Reply:
Yes that's true. Just need to add another cast here.